apps/asap-server/src/entities/research-output.ts

```diff
@@ -1,11 +1,16 @@
 import { ResearchOutputResponse } from '@asap-hub/model';
-import { GraphqlResearchOutput, RestResearchOutput } from '@asap-hub/squidex';
+import {
+  GraphqlResearchOutput,
+  GraphqlTeam,
+  RestResearchOutput,
+} from '@asap-hub/squidex';
 
 import { parseDate } from '../utils/squidex';
+import { parseGraphQLUser } from './user';
 
 export const parseResearchOutput = (
```

Comment:
Since this is no longer used, can we remove it?

Reply:
but we are:
https://github.com/yldio/asap-hub/blob/086f7d63072bd41cebe9fabf09db79041b64f84e/apps/asap-server/src/controllers/research-outputs.ts#L151

https://github.com/yldio/asap-hub/blob/086f7d63072bd41cebe9fabf09db79041b64f84e/apps/asap-server/src/entities/team.ts#L45